Cargo.lock

```diff
@@ -4084,6 +4084,7 @@ dependencies = [
  "sha3",
  "static_assertions",
  "strum 0.24.1",
+ "substrate-bn",
```

Comment:
Check, do we want this?

Reply:
i think it is a side-effect of bumping fuel-vm. the ecc operations under the hood use substrate-bn

Reply:
Yes I confirm